File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/Unntak.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.EksportUnntakCti;
import no.nav.domain.pensjon.regler.kode.InngangUnntakCti;

import java.io.Serializable;

public class Unntak implements Serializable {

    /**
     * Angir om personen har unntak eller ikke.
     */
    boolean unntak;

    /**
     * Angir type unntak.
     */
    InngangUnntakCti unntakType;

    /**
     * Unntak fra eksportforbud.
     */
    EksportUnntakCti eksportUnntak;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/Unntak.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.EksportUnntakEnum
import no.nav.pensjon.regler.domain.enum.InngangUnntakEnum
import java.io.Serializable

class Unntak : Serializable {
    /**
     * Angir om personen har unntak eller ikke.
     */
    var unntak = false

    /**
     * Angir type unntak.
     */
    var unntakTypeEnum: InngangUnntakEnum? = null

    /**
     * Unntak fra eksportforbud.
     */
    var eksportUnntakEnum: EksportUnntakEnum? = null

}